src/instruments/mettler_toledo/mt_sics.py

```diff
@@ -0,0 +1,328 @@
+        >>> in inst.mt_sics_commands
+        [["0", "I0"], ["1", "D"]]
+        """
+        timeout = self.timeout
+        self.timeout = u.Quantity(0.1, u.s)
+
+        retlist = []
+        self.sendcmd("I0")
+        while True:
+            try:
+                lst = self.read().split()
+                if lst == []:  # data stream was empty
+                    break
+                retlist.append(lst)
+            except OSError:  # communication timed out
+                break
+        self.timeout = timeout
+        av_cmds = [[it[2], it[3].replace('"', "")] for it in retlist]
+        return av_cmds
+
+    @property
+    def name(self):
+        """Get / Set balance name.
+
+        A maximum of 20 characters can be entered.
+
+        :raises ValueError: If name is longer than 20 characters.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.name = "My Balance"
+        >>> inst.name
+        'My Balance'
+        """
+        retval = " ".join(self.query("I10"))
+        return retval.replace('"', "")
+
+    @name.setter
+    def name(self, value):
+        if len(value) > 20:
+            raise ValueError("Name must be 20 characters or less.")
+        _ = self.query(f'I10 "{value}"')
+
+    @property
+    def serial_number(self):
+        """
+        Get the serial number of the balance.
+
+        :return: The serial number of the balance.
+        :rtype: str
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.serial_number
+        '123456789'
+        """
+        return self.query("I4")[0].replace('"', "")
+
+    @property
+    def tare_value(self):
+        """Get / set the tare value.
+
+        If no unit is given, grams are assumed.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.tare_value = 1.0
+        >>> inst.tare_value
+        <Quantity(1.0, 'gram')>
+        """
+        retval = self.query("TA")
+        return u.Quantity(float(retval[0]), retval[1])
+
+    @tare_value.setter
+    def tare_value(self, value):
+        value = assume_units(value, u.gram)
+        value = value.to(u.gram)
+        _ = self.query(f"TA {value.magnitude} g")
+
+    @property
+    def weight(self):
+        """
+        Get the stable weight.
+
+        :return: Stable weight
+        :rtype: u.Quantity
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.weight
+        <Quantity(1.0, 'gram')>
+        """
+        retval = self.query("S")
+        return u.Quantity(float(retval[0]), retval[1])
+
+    @property
+    def weight_immediately(self):
```

Comment:
Same for here

Reply:
oh yeah you're right 😄 hmmmm

Reply:
if `weight` is a property, then perhaps there should be something that is separately called that sets the measurement mode by which the weight measurement will be taken

Reply:
Could put in an `Enum` for `WeightMode`. Then create a property `weight_mode`, that defaults to `WeightMode.stable`, but can be set to `Weight.mode.immediately`. I'd still leave the kwarg for `zero` and `tare` though, but set it to `None` (meaning it would default to `WeightMode`. If set, the user can overwrite the `WeightMode` for those too routines, since I can see why you might want to get a weight immediately, but not tare or zero immediately...

Reply:
I think that's a reasonable path forward, and would be consistent with our other instruments

Reply:
Okay, i changed it according to the discussion. This is cleaner, thanks for the idea!